docker/zenml-server-hf-spaces.Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM zenmldocker/zenml-server:latest
```

Comment:
I'm a bit puzzled about what this file is for and how it relates to the docs.

Reply:
This is the Dockerfile used to spin up our official template on the Hub / in Spaces. You can see it in use [here](https://huggingface.co/spaces/zenml/zenml/tree/main).